src/js/components/basic-draggable/drag-drop-container.jsx

```diff
@@ -15,11 +15,15 @@ var ItemGroupForThemeUnion = React.createClass({
             if (parent[property].type === 'city') {
                 classNames += property;
             }
-
-            if (parentList.length < 2) {
-                classNames += " col-lg-4"; //If root nodes add into bootstrap column system.
+            if(parent[property].type === 'root'){
+                classNames += "dol-lg-" + 12/parentList.length;
```

Comment:
Looks like a typo dol-lg? I assume this should be col-lg

Reply:
Yep saw it